src/backend/distributed/planner/query_colocation_checker.c

```diff
@@ -252,8 +262,9 @@ SubqueryColocated(Query *subquery, ColocatedJoinChecker *checker)
  * projections. The returned query should be used cautiosly and it is mostly
  * designed for generating a stub query.
  */
-static Query *
-WrapRteRelationIntoSubquery(RangeTblEntry *rteRelation)
+Query *
+WrapRteRelationIntoSubquery(RangeTblEntry *rteRelation, List *requiredAttributes,
+							List **allTargetList)
```

Comment:
allTargetList could use some explanation in the comment

(also maybe, outerQueryTargetList)

Reply:
I guess in the context of `WraRTERelationIntoSubquery` this is not a `outerQueryTargetList` but for the caller it refers to `outerQueryTargetList`. So I would say we can call the caller as outerQueryTargetList? as in:

```c
	List *outerQueryTargetList= NIL;
	Query *subquery = WrapRteRelationIntoSubquery(rangeTableEntry, requiredAttrNumbers,
												  &outerQueryTargetList);
```